enterprise/backend/src/metabase_enterprise/serialization/v2/models.clj

```diff
@@ -15,7 +15,10 @@
    "Collection"
    "Dashboard"
    "NativeQuerySnippet"
-   "Timeline"])
+   "Timeline"
+   "Pulse"
+   "PulseChannel"
+   "PulseCard"])
```

Comment:
I think this is incorrect. Some of these lists might be doing double duty, and it's causing friction. These lists desperately need comments explaining exactly what goes into them.

`PulseChannel` and `PulseCard` are inlined on pulses, if memory serves. I'm not 100% sure if pulses are really top level `content` models, because pulses the feature is deprecated but we still use the table for dashboard subscriptions. so maybe they're inlined on dashboards?

Reply:
Took a look at exported `Pulse.*`, and it looks weird. `Pulse` is top-level (sits in `collections/.../pulses/`), and it does not include either PulseCard and PulseChannel. 

Also, if you're exporting just a single collection, the one with dashboard with notifications configured, Pulse is not going to be exported. Seems like an omission we'd like to fix?

Reply:
We should at least look into it. You can't create a proper pulse anymore, but we should make sure dashboard subs are getting serialized - assuming we actually want that. We actually might not... check with @luizarakaki . If they're excluded on purposes we should say why in a comment.

Reply:
@bshepherdson okay so we're not exporting Pulse at all. Do you think it makes sense to remove serdes methods and entity_id from Pulse* in that PR?

Reply:
Be careful, you can still create rows in the `Pulse` table, they're either dashboard subscriptions or alerts. These should still be serialized.

Reply:
Cam drew some nice diagrams here explaining how this works.
https://github.com/metabase/metabase/issues/17871

Ngoc has just deleted the "normal pulses", i.e. legacy pulses that aren't dashboard subscriptions or alerts.

Reply:
It seems like [they don't need to](https://metaboat.slack.com/archives/C01LQQ2UW03/p1700579475970899).

Reply:
Ah, brilliant

Reply:
👍